### ISSUE TITLE
Row edit button is not accessible with keyboard

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/tables#194]
The row edit button is not accessible with tab navigation as it is only shown on hover currently.

**Comment by datenangebot** [comment]
Something has gone wrong here, the fix was commited within the eslint cleanup.
As this is just a simple change, I link it here and close the issue.
Next time will be better.

https://github.com/nextcloud/tables/commit/b14309a5775513734d3a43a8f46b9b3a614e425f#diff-c870cd020ca57938df10418364a044b991edefd3e684bc3450c8ca94058b24f3R173-R175
